src/main/java/com/rcv/GuiMainController.java

```diff
@@ -33,6 +33,7 @@
 import javafx.stage.FileChooser;
 import javafx.stage.FileChooser.ExtensionFilter;
 import javafx.stage.Stage;
+import java.util.logging.Level;
```

Comment:
In this class (`GuiMainController`), I think you're going to want to replace all usages of `printToTextStatus()` with logging and delete that method entirely. You may want to recycle some of the date / formatting logic in `printToTextStatus()` into the logger.